packages/mjml-text/README.md

```diff
@@ -1,6 +1,15 @@
 ## mjml-text
 
-This tag allows you to display text in your email.
+Most important, this element places text in your email.
```

Comment:
I'm not sure about those 2 paragraphs too, I feel like we should mention that mj-text can contain any HTML you want but we expect from MJML developers to already know about HTML so the `p`/ `br` tag part is a bit overkill.

Reply:
And mj-text example alreay have an example with h1 too :)

Reply:
Fixed.